packages/paste-website/src/pages/patterns/error-state/index.mdx

```diff
@@ -0,0 +1,531 @@
+
+## Specific component guidance
+
+### When to use which component
+
+**[Alert](/components/alert)**
+
+<Alert variant="error">
+  <strong>Please update your billing information or we'll suspend your account on March 2.</strong> We couldn't renew
+  your subscription with the information we currently have. <Anchor href="#">Update billing information</Anchor>
+</Alert>
+
+- Alerts communicate a critical issue at the system level, like site-wide service disruptions, active incidents, or
+  expired billing information.
+- Alert error messages do not result from a user action.
+
+**[Callout](/components/callout)**
+
+<Callout variant="error">
+  <CalloutHeading as="h4">Missing information</CalloutHeading>
+  <CalloutText>Enter values for all required fields.</CalloutText>
+</Callout>
+
+- Callouts communicate:
+  - An error specific to a particular section of a page, or that applies to a whole page.
+  - An error summary of multiple errors on the page. When used for form validation, still include the inline error
+    messages.
+  - Use Callouts for situations like operation failures with multiple pieces of information, or a form submission
+    missing many required fields.
+- Callout error messages can result from a user action, but don’t have to.
+
+**[Toast](/components/toast)**
+
+<Toast onDismiss={() => alert('dismissed')} variant="error">
+  There was an error when deleting the profile. Please try again.
+</Toast>
+
+- Toasts communicate brief, easily comprehended error messages that are not the user’s fault, like a failed
+  connection.
+- Toast error messages usually result from a user action.
+- Toast error messages should never automatically disappear.
+
+**[Help Text](/components/help-text)**
+
+<HelpText id="email_error_help_text" variant="error">
+  Enter a valid email. Use your work email address.
+</HelpText>
+
+- Help Text communicates contextual errors, like field validation.
+- Help Text can result from a user action, but doesn’t have to.
+
+### Components with additional error guidance
+
+- [Alert](/components/alert)
+- [Badge](/components/badge)
+- [Singleselect Combobox](/components/combobox)
+- [Multiselect Combobox](/components/multiselect-combobox)
+- [Date picker](/components/date-picker)
+- [Disclosure](/components/disclosure)
+- [File picker](/components/file-picker)
+- [File uploader](/components/file-uploader)
+- [Form](/components/form)
+- [Form pill group](/components/form-pill-group)
+- [Help text](/components/help-text)
+- [Input](/components/input)
+- [Radio button group](/components/radio-button-group)
+- [Radio group](/components/radio-group)
+- [Select](/components/select)
+- [Textarea](/components/textarea)
+- [Time picker](/components/time-picker)
+- [Toast](/components/toast)
```

Comment:
Waiting to hear back from Megan on whether this list is meant to go on the doc site.